pymc/distributions/continuous.py

```diff
@@ -1714,12 +1720,18 @@ def logcdf(value, mu, sigma):
             -np.inf,
             normal_lcdf(mu, sigma, pt.log(value)),
         )
+
         return check_parameters(
             res,
             sigma > 0,
             msg="sigma > 0",
         )
 
+    def icdf(value, mu, sigma):
+        res = pt.exp(icdf(Normal.dist(mu, sigma), value))
+        res = check_icdf_value(res, value)
```

Comment:
This one isn't needed, since the same value is used in the normal icdf and has the check already

Reply:
I got confused with this part being necessary in the HalfNormal and kept it here for the LogNormal as well. Sorry about that! This is fixed now.

Reply:
No problem